lsp-methods.el

```diff
@@ -259,6 +259,26 @@ interface Range {
   (lsp--range (lsp--point-to-position start)
 	      (lsp--point-to-position end)))
 
+(defun lsp--apply-workspace-edits (workspace-edits)
+  (maphash (lambda (key value) (lsp--apply-workspace-edit key value)) (gethash "changes" edits))
```

Comment:
See https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#workspaceedit

It is accessed via the "changes" key.  As presented in emacs-lsp when the reply is sent, and working in vscode.